test/library/packages/Sort/RadixSort/flagsort.chpl

```diff
@@ -0,0 +1,33 @@
+ use BitOps;
+ use Sort;
+ use Random;
+ use flagsortMSB;
+ use Time;
+
+ config const printStats = true;
+ config const size = 1000;
+  
+
+ proc main() {
+   
+   var array:[1..size] int; 
+   fillRandom(array);
+   
+   for i in array.domain {
+     array[i] = abs(array[i]);
+   }
+
+   var t: Timer;
+   t.start();
+   radixSortMSB(array);
+   t.stop();
+ 
+   /*if printStats {
+     writeln("Time: ", t.elapsed());
+   }*/
```

Comment:
It'd be good to keep this block here. But rather then merging it commented, I'd change `printStats` to be false-by-default, and uncomment these lines.